test_dynamic_search.py

```
@@ -0,0 +1,34 @@
+import unittest
+from dynamic_search import parse_search_query
+
+
+class TestSearchParser(unittest.TestCase):
+    def test_single_field(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_single_field`

Add return type annotation: `None`

(ANN201)